docs/iidm/model/threeWindingsTransformer.md

```diff
@@ -5,23 +5,34 @@ layout: default
 
 The `com.powsybl.iidm.network.ThreeWindingsTransformer` interface is used to model a three windings power transformer.
 A three windings power transformer is connected to three voltage levels (side 1, side 2 and side 3) that belong to the
-same substation:
+same substation, usually:
 - Side 1 is the primary side (high voltage)
-- Side 2 and Side 3 can indifferently be the secondary side (medium voltage) or the tertiary side (low voltage)
-A [Ratio Tap Changer](ratioTapChanger.md) can be associated to the side 2 or the side 3 of a three windings power transformer.
+- Side 2 is the secondary side (medium voltage)
+- Side 3 is the tertiary side (low voltage)
```

Comment:
Is there a requirement for side 2 to be medium voltage or are side 2 and side 3 interchangeable?

Reply:
see previous comment.